.yarn/versions/11abead6.yml

```diff
@@ -0,0 +1,41 @@
+releases:
+  "@yarnpkg/cli": minor
+  "@yarnpkg/core": minor
+  "@yarnpkg/fslib": minor
+  "@yarnpkg/libui": minor
+  "@yarnpkg/libzip": minor
+  "@yarnpkg/plugin-interactive-tools": minor
+  "@yarnpkg/plugin-pnp": minor
+  "@yarnpkg/plugin-version": minor
+  "@yarnpkg/pnp": minor
+  vscode-zipfs: minor
```

Comment:
@paul-soporan Could this be marked as a `patch` instead so I can backport it?
I see the reason for `minor` (adding big-endian support) but one could also look at it as a `patch` (fixing big-endian support).